records_mover/records/targets/data_url.py

```diff
@@ -13,13 +15,22 @@
     from ..sources.dataframes import DataframesRecordsSource
 
 
+def sniff_records_format_from_url(url: str) -> BaseRecordsFormat:
+    if url.endswith('.parquet'):
```

Comment:
Would it ever be the case that we have .parquet.gz or .avro.gz files? I think compression is part of the file format in those formats so the answer is "no", but just verifying.